CONTRIBUTING.md

```diff
@@ -16,25 +16,31 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 ## Submit an Issue
 
 if you wish to contribute to GCToolKit we would kindly ask that you submit an issue to the issue tracker. Doing so will help with the management of the project.
- 
+
 ## Build
 
-The build requires JDK 11, and uses the Maven wrapper (`mvnw`) to help ensure reproducible builds and so we don't force you to change your system Maven install. If you prefer to build with your local Maven installation, make sure the version matches the one in the project's [.mvn/wrapper/maven-wrapper.properties](https://github.com/microsoft/gctoolkit/blob/main/.mvn/wrapper/maven-wrapper.properties) file.
+The build requires JDK 11 and uses the Maven wrapper (`mvnw`) to help ensure reproducible builds and so we don't force you to change your system Maven install. If you prefer to build with your local Maven installation, make sure the version matches the one in the project's [.mvn/wrapper/maven-wrapper.properties](https://github.com/microsoft/gctoolkit/blob/main/.mvn/wrapper/maven-wrapper.properties) file. You can also use JDK 17 or 21 by passing in `-Dmaven.compiler.release=<17|21>` as an extra property to any of the `./mvnw` commands below.
```

Comment:
Probably of no consequence, but if my JDK is 21 and I'm building from the command line, I could give (almost) any release number. 19 for example. I just can't give a release number greater than the version of my JDK.

Reply:
I agree but I think I'd prefer us to only recommend LTS's